src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/Licenses.source-build-externals.json

```diff
@@ -0,0 +1,16 @@
+{
+  "files": [
+    {
+      "path": "src/application-insights/LOGGING/ThirdPartyNotices.txt",
+      "detected_license_expression": "unknown AND apache-2.0 AND mit AND bsd-new"
+    },
+    {
+      "path": "src/application-insights/WEB/ThirdPartyNotices.txt",
+      "detected_license_expression": "bsd-new AND mit AND ms-pl AND apache-2.0 AND (cc-by-3.0-us AND cc-by-3.0 AND mit) AND ms-net-library AND ms-rl"
```

Comment:
[`ms-net-library` is a proprietary license](https://github.com/nexB/scancode-toolkit/blob/c1575930c410c27d1f0f95245e6eff54e0d60950/src/licensedcode/data/licenses/ms-net-library.LICENSE#L2). IMO, it does not belong in the baseline. We should classify it as a false positive or get it fixed.

Reply:
This is being addressed by https://github.com/dotnet/source-build-externals/pull/220